static/app/components/events/searchBar.tsx

```diff
@@ -304,7 +295,7 @@ function SearchBar(props: SearchBarProps) {
               )
             : Object.assign({}, STATIC_FIELD_TAGS_WITHOUT_TRACING);
 
-    Object.assign(combinedTags, tagsWithKind, STATIC_SEMVER_TAGS, supportedTags);
+    Object.assign(combinedTags, tagsWithKind, STATIC_FIELD_TAGS, STATIC_SEMVER_TAGS);
```

Comment:
also, `STATIC_FIELD_TAGS` is already assigned to combinedTags, why do we to assign it again here?

Reply:
I just reverted the code to what it was before

Reply:
but it does make sense! let me update it